app/views/meals/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= simple_form_for(Meal.new, as: FoodItem.first) do |f| %>
```

Comment:
views shouldn't contain activemodel/activerecord logic. 
And what's that at all - form for meal, but as food item.